api/v1/middleware/auth.js

```diff
@@ -51,7 +51,7 @@ module.exports = (req, res, next) => {
               return next(new errors.InvalidHeaderError(message, source));
             }
 
-            logger.info('Impersonation: %d %d at %s with %s %s',
+            logger.debug('Impersonation: %d %d at %s with %s %s',
```

Comment:
We should make sure production has its log level set to debug. This is information we want to see